### ISSUE TITLE
Setting unknown property: benf\neo\records\BlockType::groupId

### ISSUE THREAD

**Issue by maxstrebel** [spicywebau/craft-neo#555]
#### Description

When I apply my project Config with a newly created NEO field, it throws the error `error: Setting unknown property: benf\neo\records\BlockType::groupId`.

#### Steps to reproduce
1. Have a database dump without NEO installed
2. Install NEO
3. Create a neo field
4. Import dump without neo installed
5. Run `project-config/apply`

#### Other information
- Neo version: 2.13.3
- Craft version: 3.7.38
- (if multi-site) What is the affected Neo field's propagation method:
- Is eager-loading used? No

<img width="938" alt="image" src="https://user-images.githubusercontent.com/910594/162620139-57877982-7988-4583-a2c5-742256e391eb.png">

**Comment by ttempleton** [comment]
I'm unable to reproduce this, but I'm wondering if your `neoblocktypes` table has the `groupId` column after you try the `craft project-config/apply`? If not, does your `migrations` table have a row with the name `m220228_081104_add_group_id_to_block_types`?

**Comment by maxstrebel** [comment]
So I dug deeper and the following happened: the project I took might have used NEO in the past. So they have NEO tables on the current production database, although NEO is not installed and there are no fields.

When I imported the production Database, it seems not to apply the migration, I do not know why. But the column does not exist, also after running the migrations on the Command line. That is the reason I cannot apply.

After I manually deleted the NEO tables on production, I can run the project-config. I do not know if this still something worth investigating. If not, I can close the ticket. My problem is solved.
